packages/clay-core/src/typography/Text.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+ import classNames from 'classnames';
+ import React from 'react';
+
+ export type TextLevel =
+	| 1
+	| 2
+	| 3
+	| 4
+	| 5
+	| 6
+    | 7
+    | 8
+	| 9
+	| 10
+	| 11;
+
+export type WeightFont =
+	| 'lighter'
+	| 'light'
+	| 'normal'
+	| 'semi-bold'
+	| 'bold'
+	| 'bolder';
+
+export type DisplayType =
+	|  null
+	| 'primary'
+	| 'secondary'
+	| 'muted'
+	| 'success'
+	| 'warning'
+	| 'danger'
+	| 'info'
+
+
+interface IBaseProps extends React.BaseHTMLAttributes<HTMLSpanElement> {
+    /**
+     * Determine the color text.
+     */
+     
+    displayType?: DisplayType;
+
+    /**
+     * Set the text in italic style.
+     */
+    italic?: boolean;
+
+    /**
+     * Number to set the heading level.
+     */
+    level?: TextLevel;
```

Comment:
I think for this component the `size` would be much more common and also the default value for this prop should be the base size of 16px, in this case, `text-4`.